### DISCUSSION TITLE
Trajectory estimation of a ring into a pole.

### DISCUSSION THREAD

**Discussion by Arnav1145** [community/community#31348] [Other Feature Feedback, Questions, & Ideas]
Hello everyone,
I want to know how can a trajectory of a ring being thrown into a pole (**Ring Toss Game**)  can be decided considering all the forces acting on it? 
Can a **machine learning approach** help **predict the exact angle & force** applying on to the ring for reaching inside a pole ?

![image](https://user-images.githubusercontent.com/74757456/187430775-2fb2f793-9b8a-436f-b288-93847052bed2.png)

**Comment by jfullstackdev** [comment]
I'm very sure there are similar open source projects or if there is none, fragments of it maybe, you may want to search developers of same interest too

take for example this public search

https://github.com/topics/projectile-motion
